config/couponables.php

```diff
@@ -46,4 +46,20 @@
 
     'service' => \MichaelRubel\Couponables\Services\CouponService::class,
 
+    /*
+    | Rounding precision if you use calculations.
+    |
+    | Default: 2
+    */
+
+    'round' => 2,
```

Comment:
I'm wondering if rounding is acceptable for this kind of operation. 🤔
We should consider using the [`Number`](https://github.com/michael-rubel/laravel-value-objects/blob/main/src/Collection/Primitive/Number.php) instead of floats but in this case, the package will be dependent on [`Value Objects Package`](https://github.com/michael-rubel/laravel-value-objects)